### PR TITLE
DOCSP-38056: Add docs release notes for w/e March 29

### PR DESCRIPTION
**Pull request by cbullinger**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-38056

### CODE REVIEW

File: docs-release-notes.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# March 29, 2024
+
+## Flutter SDK
+
+- Throughout Flutter docs: Update any `.g.dart` file extensions to the new `.realm.dart` extension.
```
Comment (lindseymoore) [inline docs-release-notes.md:5]: Typically, we put updates dealing with docs throughout the SDK at the end of the section!

**Review by lindseymoore — APPROVED** [review]
LGTM with small change!

File: docs-release-notes.md (hunk 1 above)
Comment (cbullinger) [inline docs-release-notes.md:5]:
```suggestion
```

File: docs-release-notes.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+- Install: Add note regarding the breaking change to how data model class files generate, and update "Prerequisites" section with updated version and platform requirements.
+- Realm Database/Model Data:
+  - Define a Realm Object Schema: Add note regarding the breaking change to how
+    data model class files generate.
+  - Data Types: Update code examples with `dynamic.getList()`, `dynamic.getSet()`, and `dynamic.getMap()` methods.
+- Upgrade to Flutter SDK v2.0.0: New page detailing the breaking changes to SDK version 2.0.0 and how to upgrade an app from an earlier version.
```
Comment (cbullinger) [inline docs-release-notes.md:11]: - Throughout Flutter docs: Update any `.g.dart` file extensions to the new `.realm.dart` extension.

**Pull request merged**